src/main/java/seedu/address/storage/StorageManager.java

```diff
@@ -73,7 +73,7 @@ public void saveAddressBook(ReadOnlyAddressBook addressBook) throws IOException
     @Override
     public void saveAddressBook(ReadOnlyAddressBook addressBook, Path filePath) throws IOException {
         logger.fine("Attempting to write to data file: " + filePath);
-        logger.log(Level.WARNING, "Updating the file manually may result in unexpected behaviour and/or data loss");
+        logger.log(Level.INFO, "Updating the file manually may result in unexpected behaviour and/or data loss");
```

Comment:
Why this change? What is the difference here?

Reply:
This was adjusted to not cause alarm to any developers when looking through logs, as this function call happens quite often. Furthermore, the string acts as more of a tip rather than a warning at this specific point as the user interaction does not have anything to do with the warning itself.